proposals/solid-oidc/primer.bs

```diff
@@ -0,0 +1,745 @@
+
+Once the OP has confirmed that everything checks out and all the tokens are generated, it sends a response with the access token in the body:
+
+Response (content-type: application/json)
+```
+{
+  "access_token": "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJ3ZWJpZCI6Imh0dHBzOi8vYWxpY2UuY29vbHBvZC5leGFtcGxlL3Byb2ZpbGUvY2FyZCNtZSIsImlzcyI6Imh0dHBzOi8vc2VjdXJlYXV0aC5leGFtcGxlIiwiYXVkIjoic29saWQiLCJjbmYiOnsiamt0IjoiOVhtd0s4bVEzSDUtUG56QXQzbEZIeldCV192NVFoWXluZXpiYml0NGtDOCJ9LCJjbGllbnRfaWQiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL3dlYmlkI3RoaXMiLCJqdGkiOiJmZDBhMTM1My0yNWYzLTRjN2UtOGY4Yi1jMTQyNjRmMWMxMmUiLCJpYXQiOjE2MDMzODUyNjEsImV4cCI6MTYwMzM4NTg2MX0.HIIYiovILPvsdkD3s3xomR1MkA_Ir8Mx_C-eHxbWEnw9Z2rv2rOWbvMPZ--BqH3qVSupgYTZZsTDbZXA8giCLA",
+  "expires_in":	300,
+  "token_type":	"DPoP",
+  "id_token": "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJhdWQiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL3dlYmlkI3RoaXMiLCJ3ZWJpZCI6Imh0dHBzOi8vYWxpY2UuY29vbHBvZC5leGFtcGxlL3Byb2ZpbGUvY2FyZCNtZSIsImlzcyI6Imh0dHBzOi8vc2VjdXJlYXV0aC5leGFtcGxlIiwianRpIjoiODQ0YTA5NWMtOWNkYi00N2U1LTk1MTAtMWRiYTk4N2MwYTVmIiwiaWF0IjoxNjAzMzg2NDQ4LCJleHAiOjE2MDMzODcwNDh9.T306vT8dmn9gQIMEdG92AM4WRnrhqWZTfDpovwqZ6Zn0mK9yxj0iOVGqXD4CW8-tzDTitNwEGorAo85atL0Oeg",
+  "refresh_token":	"eyJhbGciOiJub25lIn0.eyJqdGkiOiJhNzhiNDllZi03MWM1LTQ5ODUtYTUwYy01ZWYzYWVmMGZkOGYifQ."
+  "scope":	"openid webid offline_access"
+}
+```
+ - `access_token": "eyJhbGciOiJ..."`: The access token we generated. The client will use this to authenticate with the server.
+ - `"expires_in":	300`: Tells the client that the access token will expire in 300 seconds (5 minutes)
+ - `"token_type":	"DPoP"`: Tells the client that the token type is DPoP
+ - `"id_token": "eyJhbGciOiJFU..."`: The id token we generated. The client will use this to extract information like the user's WebId.
+ - `"refresh_token":	"eyJhbGciOiJ..."`: The refresh token. The client will use this to get a new access token when its current one expires.
+ - `"scope":	"openid webid offline_access"`: The scopes that were used.
+
+### Sending a Request
+
+<img src="primer-request.png" />
+
+#### 1. An AJAX request is initiated
+
+In this example, Alice has logged into https://decentphotos.example and has completed the authentication steps above. She wants to make a request to Bob's Pod to get a photo album information at `https://bob.otherpod.example/private/photo_album.ttl`. Bob has previously granted access to Alice but has not granted access to anyone else.
+
+#### 2. Creates a DPoP header token
+
+Before we send a request, we need to generate a DPoP header token. A new DPoP token must be generated every time a request is made.
+
+Generating a DPoP token is done the same way we did it in the authentication section. Our token could look like the following (you can decode the token using https://jwt.io):
```

Comment:
Might be worth mentioning that a client needs to use the same keypair as before.